code/controllers/subsystem/human_ai.dm

```diff
@@ -15,7 +15,7 @@ SUBSYSTEM_DEF(human_ai)
 	var/list/squad_id_dict = list()
 
 	/// The current highest ID of any squad
-	var/highest_squad_id = 0
```

Comment:
why

Reply:
now highest_squad_id is increased AFTER generating new squad name, so in case we don't wanna have a squad "0" in our panel - we'll have to change it to 1